src/ape_accounts/_cli.py

```diff
@@ -157,6 +157,17 @@ def _import(cli_ctx, alias, import_from_mnemonic, custom_hd_path):
     )
 
 
+@cli.command(short_help="Export an account private key")
+@ape_cli_context()
+@existing_alias_argument()
```

Comment:
So this `existing_alias_argument` decorator takes an `account_type` kwarg. I think we should set that to the `KeyfileAccount`. Else, it lets me give it a ledger / trezor account or even a starknet one it fails when trying to open the file because it doesn't exist. Those accounts should not even be options.

Reply:
I am unsure what to do to fix this, but it could alternatively try and then fail with the error that it's not an exportable key?